internal/config/system.go

```diff
@@ -245,4 +251,9 @@ type ModelServerPods struct {
 
 	// ImagePullSecrets is a list of references to secrets in the same namespace to use for pulling any of the images
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
+
+	// ModelPodPatches is a list of patches to apply to the model pod template.
+	// This is a JSON Patch as defined in RFC 6902.
+	// https://datatracker.ietf.org/doc/html/rfc6902
+	ModelPodPatches []Patch `json:"podPatches,omitempty"`
```

Comment:
Rename to `JSONPatches` (json: `jsonPatches`). This is within the context of `ModelServerPods` ... so we don't need the duplicate "Pod" part of the name. The JSON part specifies what kind of patch this is.